community/front-end/ofe/infrastructure_files/workbench_tf/google/main.tf

```diff
@@ -110,7 +110,7 @@ data "google_compute_subnetwork" "default" {
 module "vpc_ai_notebook" {
   count   = var.create_network ? 1 : 0
   source  = "terraform-google-modules/network/google"
-  version = "~> 3.0"
+  version = "~> 7.0"
```

Comment:
same here, this seems like a pretty big jump, do we know if it works for sure?

Reply:
I've run all of the integration tests, but I'm not sure about how much of this module it covers.  Maybe after talking to the team tomorrow, I'll have a better idea of how to test this.